src/roman_datamodels/maker_utils/_common_meta.py

```diff
@@ -254,7 +254,7 @@ def mk_velocity_aberration(**kwargs):
     vab = stnode.VelocityAberration()
     vab["ra_reference"] = kwargs.get("ra_reference", NONUM)
     vab["dec_reference"] = kwargs.get("dec_reference", NONUM)
-    vab["scale_factor"] = kwargs.get("scale_factor", NONUM)
+    vab["scale_factor"] = kwargs.get("scale_factor", 1)
```

Comment:
Are these expected to be reasonable? I assumed they were intentionally nonsense values that the pipeline replaces.

Reply:
The reason to give a reasonable default here is that we are going to start actually using this value.  The scale factor makes a pretty small effect (fraction of a pixel or so), so you don't really want things to explode if people don't set it.  Something like ra/dec doesn't have a reasonable default that's right to within a fraction of a pixel all of the time, so it's reasonable to put a crazy number there, but lots of people won't want to think about the scale factor and my preference is not to force them to.

Reply:
Thanks. It's helpful to know that these are sometimes valid defaults.